docs/source/en/training/create_dataset.mdx

```diff
@@ -0,0 +1,84 @@
+# Create a dataset for training
+
+There are many datasets on the [Hub](https://huggingface.co/datasets?task_categories=task_categories:text-to-image&sort=downloads) to train a model on, but if you can't find one you're interested in or prefer to use your own, you can create a dataset with the 🤗 [Datasets](hf.co/docs/datasets) library. The dataset structure depends on the task you want to train your model on. Some common formats include a directory of only images for tasks like unconditional image generation or images with corresponding text captions for tasks like text-to-image generation.
+
+This guide will show you two ways to create your own dataset to finetune on:
+
+- provide your own folder of images to the `--train_data_dir` argument
+- upload your dataset to the Hub and pass the dataset repository id to the `--dataset_name` argument.
+
+<Tip>
+
+💡 Learn more about how to create an image dataset for training in the [Create an image dataset](https://huggingface.co/docs/datasets/image_dataset) guide. 
+
+</Tip>
+
+## Provide the dataset as a folder
+
+For unconditional generation, you can provide your own dataset as a folder of images. The training script uses the [`ImageFolder`](https://huggingface.co/docs/datasets/en/image_dataset#imagefolder) builder from 🤗 Datasets to automatically build a dataset from the folder. Your directory structure should look like:
+
+```bash
+data_dir/xxx.png
+data_dir/xxy.png
+data_dir/[...]/xxz.png
+```
+
+Pass the path to the folder containing the images to the `--train_data_dir` argument and launch the training:
+
+```bash
+accelerate launch train_unconditional.py \
+    --train_data_dir <path-to-train-directory> \
+    <other-arguments>
+```
+
```

Comment:
Maybe add a note on "For rest of the details on the training, please refer to ... script."